src/main/scala/cromwell/binding/AstTools.scala

```diff
@@ -18,8 +19,7 @@ object AstTools {
     def findTerminals(): Seq[Terminal] = AstTools.findTerminals(astNode)
     def findTopLevelMemberAccesses(): Iterable[Ast] = AstTools.findTopLevelMemberAccesses(astNode)
     def sourceString(): String = astNode.asInstanceOf[Terminal].getSourceString
-    
-    def wdlType: WdlType = {
+    def wdlType(wdlSyntaxErrorFormatter: WdlSyntaxErrorFormatter): WdlType = {
```

Comment:
Yeah especially since the formatter is only really needed if something goes awry anyway.  Agreed this can be tech debt, just wanted to note it.  :smile: